### PR TITLE
Dev4

### PR DESCRIPTION
**Pull request by dummm1**
Please merge me. Thanks

### CODE REVIEW

**Comment by zealzel** [comment]
Today is Friday.

**Comment by zealzel** [comment]
OK. This is good.

File: new.txt [hunk 1]
```diff
@@ -13,3 +13,8 @@ dev3 {
   line1
   line2
 }
+
+dev4 {
```
Comment (zealzel) [inline new.txt:17]: What is this function for?

Comment (dummm1) [reply new.txt:17]: This is just for the noob and boon

Comment (zealzel) [reply new.txt:17]: ok. I got it.

**Review by zealzel — COMMENTED** [review]
Nice. It looks good.

File: new.txt [hunk 2]
```diff
@@ -13,3 +13,8 @@ dev3 {
...
+  line1
+  line2
```
Comment (zealzel) [inline new.txt:19]: I think line2 should be xxxx

Comment (zealzel) [reply new.txt:19]: comment2

Comment (zealzel) [reply new.txt:19]: comment3

Comment (zealzel) [reply new.txt:19]: comment4

Comment (zealzel) [reply new.txt:19]: comment5

Comment (zealzel) [reply new.txt:19]: comment6

**Review by zealzel — COMMENTED** [review]
COMMENT FINISHED

**Review by zealzel — APPROVED** [review]

**Pull request merged**